assets/scss/base.scss

```diff
@@ -361,73 +299,12 @@ a.v-btn {
 }
 // Common button end
 
-.h-divider {
-  margin: 0;
-  border-color: #e2e8ee;
+.theme--light.v-divider.h-divider {
+  border-width: 0 0 3px 0;
```

Comment:
The previous code declared a divider with a 3px top, right, bottom and left border :(

Now we only have a single border declared.